<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnergyValue</class>
 <widget class="QDialog" name="EnergyValue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>399</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Енергетичні величини</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>314</width>
     <height>378</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="initial_energy_label">
          <property name="text">
           <string>Початкова енергія вузла (кДж)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="initial_energy_spin_box">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>3000</number>
          </property>
          <property name="value">
           <number>55</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="receive_power_label">
          <property name="text">
           <string>Потужність в режимі прийому (мВт)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="receive_power_spin_box">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="value">
           <number>52</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="transfer_power_label">
          <property name="text">
           <string>Потужність в режимі передачі (мВт)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="transfer_power_spin_box">
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>200</number>
          </property>
          <property name="value">
           <number>45</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="active_power_label">
          <property name="text">
           <string>Потужність в активному режимі (мВт)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="active_power_spin_box_2">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>20</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="sleep_power_label">
          <property name="text">
           <string>Потужність в режимі сну (мкВт)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sleep_power_spin_box">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="value">
           <number>90</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="random_button">
      <property name="text">
       <string>Згенерувати випадкові дані</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QDialogButtonBox" name="dialog_button_box">
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
      <property name="centerButtons">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
